wakuv2/missing_messages.go

```diff
@@ -0,0 +1,257 @@
+package wakuv2
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+	"slices"
+	"time"
+
+	"github.com/libp2p/go-libp2p/core/peer"
+	"google.golang.org/protobuf/proto"
+
+	"go.uber.org/zap"
+
+	gethcommon "github.com/ethereum/go-ethereum/common"
+
+	"github.com/waku-org/go-waku/waku/v2/protocol"
+	"github.com/waku-org/go-waku/waku/v2/protocol/store"
+
+	"github.com/status-im/status-go/logutils"
+	"github.com/status-im/status-go/wakuv2/common"
+
+	"github.com/waku-org/go-waku/waku/v2/protocol/pb"
+)
+
+const maxContentTopicsPerRequest = 10
```

Comment:
Better to increase this to 50?

Reply:
This limit comes from https://github.com/waku-org/nwaku/blob/master/waku/waku_archive/archive.nim#L144-L145

Reply:
Ah, was not aware that 10 is the limit. Odd though, could it be because of SQL query size?
Anyways we have a page limit separately so why such a low limit on contentTopics. Maybe @Ivansete-status has an idea.